### ISSUE TITLE
Calling MC commands with M-x

### ISSUE THREAD

**Issue by paldepind** [magnars/multiple-cursors.el#251]
When I call an MC command with M-x I'm asked to enter a new command. I expected the commands to work just like when invoked from a key binding.

**Comment by magnars** [comment]
Yep. See the "Known limitations" section.

**Comment by paldepind** [comment]
Thank you for the quick reply. I don't think it addresses what I'm talking about. Let me explain myself better.

I do something like the following `M-x mc/mark-all-like-this`. I expect this to work exactly like invoking `mc/mark-all-like-this` from a key binding. Instead I am immediately asked to enter a second command. If I enter a command it is applied to the first match and then I'm asked for another one and so on. If I press `C-g` I'm left in a weird state where only the first cursor does anything.

**Comment by magnars** [comment]
`M-x` is a weird one, and I haven't been able to figure out how to work with it properly - in either case. It needs keybindings. I guess the readme could have been clearer about that!

**Comment by paldepind** [comment]
I think it should. When I try out a new package I won't add bindings immediately. I'll generally try the different commands with `M-x`.

**Comment by nispio** [comment]
I can't reproduce the described behavior with `M-x mc/mark-all-like-this`.  @paldepind, is `multiple-cursors-mode` already active when you invoke this?  If so, I don't think that is the intended usage of the command.

**Comment by shackra** [comment]
I experience this but when using some mc commands with [Hydra](https://github.com/abo-abo/hydra/)

**Comment by nispio** [comment]
I wonder if this is actually a bad interaction with Helm.  See Issue #284 and the solution there.

tl;dr  Make sure that `helm-M-x` is in your `mc/cmds-to-run-once` list and not in your `mc/cmds-to-run-for-all` list.

**Comment by nispio** [comment]
I'm going to assume that this issue is related to Helm, and it is solved by #284 .  If that doesn't fix the problem, and you have a means of reproducing the issue from `emacs -q` bring it back here.
